.github/workflows/util/execute_and_retry.sh

```diff
@@ -0,0 +1,25 @@
+#!/bin/bash
+
+execute_and_retry () {
+  retry_counter=0
+  max_retry=$1
+  while [ $retry_counter -lt $max_retry ]; do
+   deployment_failed=0
```

Comment:
This is no longer specific to the deployment code, so make it clear this tracks attempt failures.
```suggestion
   attempt_failed=0
```